public/client.js

```diff
@@ -97,7 +97,19 @@ $(function () {
       $('#chatApprovalButtons').show();
     }
   })  
-  
 
+  // Chat Logic
+  $('#enterChatButton').click(function(){
```

Comment:
Note: This was already implemented in the Lobby screen back end. I'll need to resolve merge conflicts for this